examples/ethane.toml

```diff
@@ -0,0 +1,41 @@
+# run via:
+# cargo run ethane.toml --release
```

Comment:
That's true. I think having a lot of examples is a good thing for now. We can sort out examples showing features later.